### PR TITLE
SE-13063: Fixes not being able to save configurations more than once.

### PR DESCRIPTION
**Pull request by fhaScireum**
This error only manifests itself in Chrome browser (possibly other chromium browsers), as the javascript is not executed due to `event.stopPropagation();`. Firefox for instance was never affected. This issue was introduced by https://github.com/scireum/sirius-web/pull/1286

Fixes: [SE-13063](https://scireum.myjetbrains.com/youtrack/issue/SE-13063)

### CODE REVIEW

**Comment by mko-sci** [comment]
What is meant by more than once?
Once per page visit?
Once (total since application deployment)?

**Review by mko-sci — APPROVED** [review]

**Review by jmuscireum — APPROVED** [review]

**Review by mkeckmkeck — APPROVED** [review]

**Review by oolscireum — APPROVED** [review]

File: src/main/resources/default/templates/biz/tenants/tenant-config.html.pasta [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 <i:invoke template="/templates/biz/tenants/tenant.html.pasta" tenant="tenant" controller="controller" page="config">
     <t:codeEditor id="editor" mode="json">@tenant.getTenantData().getConfigString()</t:codeEditor>
-    <t:formBar backButton="false"/>
+    <t:formBar backButton="false" singleClick="false"/>
```
Comment (andyHa) [inline src/main/resources/default/templates/biz/tenants/tenant-config.html.pasta:6]: I think we should rather limit the single-click handling to pages where a form actually exists...:
https://github.com/scireum/sirius-web/blob/4836f26736699091181787d5e4591c25cf9e36b1/src/main/resources/default/assets/tycho/scripts/form.js.pasta#L2C36-L19C8

Comment (andyHa) [reply src/main/resources/default/templates/biz/tenants/tenant-config.html.pasta:6]: We also should perform a null-check here:
https://github.com/scireum/sirius-web/blob/4836f26736699091181787d5e4591c25cf9e36b1/src/main/resources/default/assets/tycho/scripts/form.js.pasta#L17C1-L17C1

**Review by andyHa — CHANGES_REQUESTED** [review]

File: src/main/resources/default/templates/biz/tenants/tenant-config.html.pasta (hunk 1 above)
Comment (fhaScireum) [reply src/main/resources/default/templates/biz/tenants/tenant-config.html.pasta:6]: > We also should perform a null-check here: https://github.com/scireum/sirius-web/blob/4836f26736699091181787d5e4591c25cf9e36b1/src/main/resources/default/assets/tycho/scripts/form.js.pasta#L17C1-L17C1

We already have a null check here: https://github.com/scireum/sirius-web/blob/4836f26736699091181787d5e4591c25cf9e36b1/src/main/resources/default/assets/common/core.js.pasta#L375



> I think we should rather limit the single-click handling to pages where a form actually exists...: https://github.com/scireum/sirius-web/blob/4836f26736699091181787d5e4591c25cf9e36b1/src/main/resources/default/assets/tycho/scripts/form.js.pasta#L2C36-L19C8

I feel like there could be instances where data i submitted "indirectly" (outside any form like it is the case here) where we want to only be able to submit the data once. I think it's cleaner when singleClick == false always means that the submit action is only triggered once.

Comment (andyHa) [reply src/main/resources/default/templates/biz/tenants/tenant-config.html.pasta:6]: If there is no form, there is no page reload and having single click == true will always screw the behaviour. On the other hand, if there is a page-reload, why not use a simple form anyway?

TL;DR - no form -> no single-click enforcing seems always right...

**Comment by fhaScireum** [comment]
Closed via alternative solution in https://github.com/scireum/sirius-web/pull/1290
